modules/core/src/main/java/org/apache/ignite/internal/commandline/CommandHandler.java

```diff
@@ -193,6 +194,9 @@
     /** */
     private static final String CMD_SKIP_ZEROS = "--skipZeros";
 
+    /** Cache kind. */
+    private static final String CACHE_KIND = "--cacheFilter";
```

Comment:
Please rename variable and update comment.

Reply:
done.